posts/2023/birth-quick-form/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Birth Quick Form
+date: 2023-05-22
+author: Michael Stenta
+slug: 2023/birth-quick-form
+---
+
+# Birth Quick Form
+
+The farmOS [v2.1.0 release](https://github.com/farmOS/farmOS/releases/tag/2.1.0)
+includes a new and improved *Birth Quick Form* for quickly and easily recording
+animal births. This feature existed in farmOS v1, and has now been ported to v2.
+
+See the video below for a quick demonstration, and read on for more details.
+
+<video autoplay control loop style="width: 100%; height: auto;">
```

Comment:
`style="width: 100%; height: auto;"`
Since these inline styles have to be repeated for every video element used in any blog post, it might be better to instead add them globally in the website styles https://github.com/farmOS/farmOS.org/blob/main/src/theme.js#L20-L34

Reply:
I had the same thought @wotnak. Perhaps I'll open a separate PR for that and depending on which of these gets merged first we can update this code.

Reply:
https://github.com/farmOS/farmOS.org/pull/83